nodes/mainnet.md

```diff
@@ -33,6 +33,49 @@ functional, users may encounter occasional instability or reduced performance.
 
 <MainnetVersionTags/>
 
+## Network parameters
+
+Full network parameters, such as [max bytes](https://github.com/celestiaorg/celestia-app/blob/23d13d4de41631dc3c52f7d94fc214e44d03962d/specs/src/specs/params.md?plain=1#L30),
+can be found in the
+[celestia-app specifications](https://celestiaorg.github.io/celestia-app/specs/params.html)
+or from `max_bytes` in
+[Mainnet Beta validator's consensus parameters](https://rpc.lunaroasis.net/consensus_params).
```

Comment:
[clarification] there are more network parameters than consensus parameters so the second part of this sentence seems a bit misleading. Proposal to either remove "Full" from the beginning or the last phrase which links to consensus params.

```suggestion
Full network parameters, such as [max bytes](https://github.com/celestiaorg/celestia-app/blob/23d13d4de41631dc3c52f7d94fc214e44d03962d/specs/src/specs/params.md?plain=1#L30),
can be found in the
[celestia-app specifications](https://celestiaorg.github.io/celestia-app/specs/params.html).
```

Reply:
thanks for the review! good call. I am going to stick with this suggestion, as it simplifies things for the reader.